Products/ZenModel/migrate/addZingConfigToZep.py

```diff
@@ -0,0 +1,395 @@
+                                name="p98",
+                                description="98th percentile time in processEvent",
+                                counter=False,
+                                unit="seconds"
+                            ),
+                            Metric(
+                                ID="zeneventserver.zing.processEvent.p99",
+                                name="p99",
+                                description="99th percentile time in processEvent",
+                                counter=False,
+                                unit="seconds"
+                            ),
+                            Metric(
+                                ID="zeneventserver.zing.processEvent.p999",
+                                name="p99.9",
+                                description="99.9th percentile time in processEvent",
+                                counter=False,
+                                unit="seconds"
+                            ),
+                            Metric(
+                                ID="zeneventserver.zing.processEvent.stddev",
+                                name="stddev",
+                                description="stddev time in processEvent",
+                                counter=False,
+                                unit="seconds"
+                            ),
+                            Metric(
+                                ID="zeneventserver.zing.processEvent.m15_rate",
+                                name="15MinuteRate",
+                                description="15 minute processEvent rate",
+                                counter=False,
+                                unit="evt/sec"
+                            ),
+                            Metric(
+                                ID="zeneventserver.zing.processEvent.m1_rate",
+                                name="1MinuteRate",
+                                description="1 minute processEvent rate",
+                                counter=False,
+                                unit="evt/sec"
+                            ),
+                            Metric(
+                                ID="zeneventserver.zing.processEvent.m5_rate",
+                                name="5MinuteRate",
+                                description="5 minute processEvent rate",
+                                counter=False,
+                                unit="evt/sec"
+                            ),
+                            Metric(
+                                ID="zeneventserver.zing.processEvent.mean_rate",
+                                name="meanRate",
+                                description="mean processEvent rate",
+                                counter=False,
+                                unit="evt/sec"
+                            )
+                        ]
+                    )
+                )
+
+            do_commit = True
```

Comment:
I think `do_commit = True` should be moved to lines 76 and 238, so we only commit when things are changed